.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
       cache:
         - ccache
       before_install:
-        - docker pull rajathprasad/rocksplicator-build:latest
+        - docker pull gopalrajpurohit/rocksplicator-build:librdkafka_1_4_0
```

Comment:
My personal preference is to tag is `latest` instead of `build:librdkafka_1_4_0`. That's the convention I've seen.

Reply:
latest tag is moving target which in normal case is fine. Here since we are upgrading the librdkafka significantly... and is the major change, I feel tagging it likewise would be more appropriate....